Description:Fix memory leak in acknowledge handling.
 Cherry-picked from the 4.3.18 upstream release.
Origin: http://sourceforge.net/p/xymon/code/7488/
Applied-Upstream: http://sourceforge.net/p/xymon/code/7488/
Reviewed-by: Axel Beckert <abe@debian.org>

Index: xymon/xymond/xymond.c
===================================================================
--- xymon.orig/xymond/xymond.c	2015-02-13 15:38:17.000000000 +0100
+++ xymon/xymond/xymond.c	2015-02-13 15:40:05.000000000 +0100
@@ -2365,6 +2365,8 @@
 	if (zombie->dismsg) xfree(zombie->dismsg);
 	if (zombie->ackmsg) xfree(zombie->ackmsg);
 	if (zombie->grouplist) xfree(zombie->grouplist);
+	if (zombie->lastchange) xfree(zombie->lastchange);
+	if (zombie->testflags) xfree(zombie->testflags);
 	flush_acklist(zombie, 1);
 	xfree(zombie);
 	dbgprintf("<- free_log_t\n");
